pkg/storage/driver/secrets.go

```diff
@@ -35,8 +35,12 @@ import (
 
 var _ Driver = (*Secrets)(nil)
 
-// SecretsDriverName is the string name of the driver.
-const SecretsDriverName = "Secret"
+const (
+	// SecretsDriverName is the string name of the driver.
+	SecretsDriverName = "Secret"
+	// ListPaginationLimit is the number of Secrets we fetch in a single API call.
+	ListPaginationLimit = int64(300)
```

Comment:
Is this something we should make configurable for commands using this API, such as `helm list`? Alternatively, should this be a field under the secrets driver, defaulting to this value if none is provided in the constructor?

Reply:
Wow, I just checked my emails to see that this is still open! Its a pity that a PR opened on March 22 was only reviewed on May 6, and I had moved on from this work since then. _And the related issue has been open since 2020!_

What's the guarantee that if I/anyone carve out time for this then the reviewers/maintainers would be willing to move faster here?

Reply:
There are no guarantees. Folks contribute their time as it becomes available. Nobody's getting paid to work on Helm full time. We all just do our best to do what we can in life.